packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerModule.kt

```diff
@@ -404,11 +404,15 @@ class ImagePickerModule(
   //region LifecycleEventListener
 
   override fun onHostDestroy() {
-    mWasDestroyed = true
-    mUIManager.unregisterLifecycleEventListener(this)
```

Comment:
I think we should unregister the lifecycle listener in `onDestroy` (not it `onHostDestroy`) method.